paddle/parameter/Parameter.h

```diff
@@ -324,6 +324,7 @@ class Parameter {
   std::vector<std::shared_ptr<IParameterUpdaterHook>> updaterHooks_;
 
 public:
+  void setSharedCount(int cnt) { sharedCount_ = cnt; }
```

Comment:
I see. Are we supposed to run `git pull upstream develop && git push` here so to remove this line?